contracts/lib/lib_readme.md

```diff
@@ -31,11 +34,49 @@ These functions combine enum values into bytes for efficient storage and transfe
 - `combineEnumsToBytes3(MPC_TYPE mpcType1, MPC_TYPE mpcType2, ARGS argsType)`: Combines two `MPC_TYPE` values and an `ARGS` value into a `bytes3` value.
 - `combineEnumsToBytes4(MPC_TYPE mpcType1, MPC_TYPE mpcType2, MPC_TYPE mpcType3, ARGS argsType)`: Combines three `MPC_TYPE` values and an `ARGS` value into a `bytes4` value.
 
+### Overflow and Result Validation
+- `checkOverflow`: Validates whether an overflow has occurred in a cryptographic operation.
+- `checkRes8`, `checkRes16`, `checkRes32`, `checkRes64`: Check the result for different bit sizes (8, 16, 32, and 64 bits).
+
 ### Key Management Functions
 - `getUserKey(bytes calldata signedEK, bytes calldata signature)`: Retrieves the user's AES encryption key in encrypted format, by using the provided public key to encrypt it - signature is used to validate the account ownership.
+- `deleteUserKey`: Deletes a user’s cryptographic key.
 
 This library is designed to be used as part of a secure multi-party computation framework, facilitating the handling and combination of encrypted and signed data types.
 
+### Ciphertext Validation and Decryption
+- `validateCiphertext`: Validates the integrity of a ciphertext.
+- `decrypt`: Decrypts a given ciphertext.
+
+### Onboarding and Offboarding Users
+- `onBoard`: Adds a user to the MPC protocol, loading data from `ct` type to `gt` type.
+- `offBoard`: Removes a user from the MPC protocol, transforming data from `gt` type to `ct` type.
+- `offBoardToUser`: Transfers data ownership to a user during offboarding; transforms from `gt` type to `ct` type previously prepared by the user's AES key rather than the MPC one.
+- `offBoardCombined`: Combines offboarding actions for multiple users, offboarding to both the user's AES key and the network's AES key.
```

Comment:
'offBoardCombined': operation to transforms from gt type to ct type, same as offBoard and offBoardToUser , so that it returns the ct encrypted by the network key and by the msg.sender key